packages/dds/tree/src/simple-tree/prepareForInsertion.ts

```diff
@@ -131,14 +122,17 @@ function validateAndPrepare(
 	schemaAndPolicy: SchemaAndPolicy,
 	hydratedData: FlexTreeHydratedContextMinimal | undefined,
 	fieldSchema: TreeFieldStoredSchema,
-	mapTrees: ExclusiveMapTree[],
+	mapTrees: readonly UnhydratedFlexTreeNode[],
 ): void {
 	if (hydratedData !== undefined) {
+		// Prepare content before validating side this populated defaults using the provided context rather than the global context.
```

Comment:
I'm having trouble parsing this paragraph (but it's definitely better than no paragraph - thanks). It's trying to communicate two things?

1. That we apply the defaults (via `prepareContentForHydration`) before we validate against the schema, and
2. That we have more context available to us here than we do inside of schema validation, so it's important to apply the defaults with that context while we can

?

Reply:
Good catch. I have clarified the docs related to this as they were unintelligible.